forms/UploadField.php

```diff
@@ -1498,6 +1498,9 @@ class UploadField_SelectHandler extends RequestHandler {
 	 */
 	protected $folderName;
 
+	// Set pagnation quantity for file list field 
```

Comment:
Er, you didn't actually change it though? did you make the change locally but forget to push it up?